Service/src/request_validator/schemas/util_schema.py

```diff
@@ -51,4 +52,21 @@ def validateBuyPower(buyPower, field, errors):
          'InvalidField' : field.name + ' must be an integer greater than 1 and less than 1000000'
       })
 
-   return errors
\ No newline at end of file
+   return errors
+
+
+def validatePosInt(integer, field, errors):
+   if type(integer) != int or integer <= 0:
+      errors.append({
+         'InvalidField' : field.name + ' must be a positive integer'
+      })
+
+   return errors
+
+
+def validateTransType(transType, field, errors):
+   print("TRANSTYPE IS " + transType)
```

Comment:
Python has a logging library as its standard lib. Just `import logging` to use it. Probably better than using print statements so we can set logging levels, it's also just cleaner.

Plus, if `transType` is anything but a string (passed in incorrectly or something) this will throw a `TypeError` exception. You can avoid this by doing something like so:
```
print("TRANSTYPE IS " + str(transType)
```
or
```
print("TRANSTYPE IS", transType)
```

If you import the logging library than you can simply do something like so:
```
import logging

logger = logging.getLogger(__name__)
logger.info("hi this is an info log")
logger.warning("hi this is a warning log")
logger.error("...")
...
```

Reply:
ya I implemented a logger too, so we can just use mine. that print sholdn't be there